app/main.py

```diff
@@ -1,3 +1,46 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as config:
+        data = json.load(config)
+    fuel_price = data.get("FUEL_PRICE")
+    customers = [Customer(**customer) for customer in data["customers"]]
+    shops = [Shop(**shop) for shop in data["shops"]]
+
+    for customer in customers:
+        customer.money_count()
+
+        trips_prices = []
+        for shop in shops:
+
+            fuel_expenses = customer.car.calculate_fuel_expenses(
+                customer.calculate_distance(shop.location),
+                fuel_price
+            )
+            trip_cost = shop.calculate_trip_cost(
+                shop.calculate_product_cost(customer),
+                fuel_expenses)
```

Comment:
In case you have no arguments on the same line with an open parenthesis, your closing parenthesis should not have any arguments on the same line as well.

Reply:
Thank you. I've already reformatted it.

Reply:
Also, I suppose there is a little mistake in the test in the file with the task. When it checks your code, it should include a space after "You have bought:". But tests on GitHub may not pass, because they have this string without a space